chromium_src/chrome/browser/importer/profile_writer.cc

```diff
@@ -42,3 +42,11 @@ void ProfileWriter::AddCreditCard(const std::u16string& name_on_card,
 
   personal_data->AddCreditCard(credit_card);
 }
+
+#if BUILDFLAG(IS_ANDROID)
+namespace first_run {
+bool IsChromeFirstRun() {
+  return false;
```

Comment:
I wonder why this change is required.

Reply:
this is my recommendation at https://github.com/brave/brave-core/pull/17821/commits/979befdc9dc1abc3825bb782fdd726a05b259929#r1160595494 to get rid of patch

Reply:
It's just `IsChromeFirstRun` is used in many places, we may brake some first run functionality.

Reply:
I see it is used only once at upstream's `src/chrome/browser/importer/profile_writer.cc`:
![image](https://user-images.githubusercontent.com/24739341/233059850-f6146980-0ca6-4efb-ad5d-481cc87c8341.png)

I suppose the override of `IsChromeFirstRun` at `src/brave/chromium_src/chrome/browser/importer/profile_writer.cc` works only for 
`src/chrome/browser/importer/profile_writer.cc`

Reply:
Oh, right, I didn't realize it's for `profile_writer.cc` only